python/delta/integration_tests/unity-catalog-commit-coordinator-integration-tests.py

```diff
@@ -54,54 +63,195 @@
+
+    def test_restore(self):
+        current_timestamp = str(datetime.datetime.now())
+        self.append(MANAGED_CC_TABLE)
+        spark.sql(f"RESTORE TABLE {CATALOG_NAME}.{SCHEMA}.{MANAGED_CC_TABLE} "
+                  f"TO TIMESTAMP AS OF '{current_timestamp}'")
+        updated_tbl = read(MANAGED_CC_TABLE).toDF("id")
+        assertDataFrameEqual(updated_tbl, self.setup_df)
+
+    def test_change_data_feed(self):
+        current_timestamp = str(datetime.datetime.now())
+        self.append(MANAGED_CC_TABLE)
+        updated_tbl = read_with_cdf_timestamp(
+            current_timestamp, MANAGED_CC_TABLE).select("id").toDF("id")
+        assertDataFrameEqual(updated_tbl, create_df_with_rows([(4, ), (5, )]))
+
+    # Dataframe Writer V1 Tests #
+    def test_insert_into_append(self):
+        single_col_df = spark.createDataFrame([(4, ), (5, )], schema=["id"])
+        single_col_df.write.mode("append").insertInto(f"{CATALOG_NAME}.{SCHEMA}.{MANAGED_CC_TABLE}")
+        updated_tbl = read(MANAGED_CC_TABLE).toDF("id")
+        assertDataFrameEqual(updated_tbl, create_df_with_rows([(1, ), (2, ), (3, ), (4, ), (5, )]))
+
+    def test_insert_into_overwrite(self):
+        single_col_df = spark.createDataFrame([(5, )], schema=["id"])
+        single_col_df.write.mode("overwrite").insertInto(
+            f"{CATALOG_NAME}.{SCHEMA}.{MANAGED_CC_TABLE}", True)
+        updated_tbl = read(MANAGED_CC_TABLE).toDF("id")
+        assertDataFrameEqual(updated_tbl, create_df_with_rows([(5, )]))
+
+    def test_insert_into_overwrite_replace_where(self):
+        single_col_df = spark.createDataFrame([(5, )], schema=["id"])
+        single_col_df.write.mode("overwrite").option("replaceWhere", "id > 1").insertInto(
+            f"{CATALOG_NAME}.{SCHEMA}.{MANAGED_CC_TABLE}", True)
+        updated_tbl = read(MANAGED_CC_TABLE).toDF("id")
+        assertDataFrameEqual(updated_tbl, create_df_with_rows([(1, ), (5, )]))
+
+    def test_insert_into_overwrite_partition_overwrite(self):
+        single_col_df = spark.createDataFrame([(5,)], schema=["id"])
+        single_col_df.write.mode("overwrite").option(
+            "partitionOverwriteMode", "dynamic").insertInto(
+            f"{CATALOG_NAME}.{SCHEMA}.{MANAGED_CC_TABLE}", True)
+        updated_tbl = read(MANAGED_CC_TABLE).toDF("id")
+        assertDataFrameEqual(updated_tbl, create_df_with_rows([(5,)]))
+
+    def test_save_as_table_append_existing_table(self):
+        single_col_df = spark.createDataFrame(
+            [(4, ), (5, )], schema=StructType([StructField("id", IntegerType(), True)]))
+        single_col_df.write.format("delta").mode("append").saveAsTable(
+            f"{CATALOG_NAME}.{SCHEMA}.{MANAGED_CC_TABLE}")
+        updated_tbl = read(MANAGED_CC_TABLE).toDF("id")
+        assertDataFrameEqual(updated_tbl, create_df_with_rows([(1, ), (2, ), (3, ), (4, ), (5, )]))
+
+
+    # Setting mode to append should work, however cc tables do not allow path based access.
```

Comment:
```suggestion
    # CC tables do not allow path based access.
```

I don't think that should work

Reply:
Update the comment